### ISSUE TITLE
Schedule daily, once, etc

### ISSUE THREAD

**Issue by FRodero** [DeviceFarmer/stf#469]
**What is the issue or idea you have?**
We are creating a new group with schedule class: Daily and once. The issue begin when we try add devices to the group and list of avaliable devices is empty.

**Please provide the steps to reproduce the issue.**
1.- create group
![image](https://user-images.githubusercontent.com/17677248/158163261-ebe0f0e1-5f9f-4db9-9ff6-93d7beb91553.png)
2.- save daily schedule and click on get ready.
3.- click on devices -> avaliable devices
![image](https://user-images.githubusercontent.com/17677248/158163498-ccb1ee9c-a818-4670-8e24-cfe7aa397781.png)

**What is the expected behavior?**
If we select "standard schedule class", the list of avaliable devices show all the devices connected. So, we wait same behaviour for "daily schedule class". Is it necessary any additional step?

**Comment by denis99999** [comment]
@FRodero ,

Yes, it is normal because f you want to create a device booking group as a simple user, you have first to create a `bookable `group as the administrator user, in which you will add some devices from the `Common standard` group (see the `GroupFeature.pdf` documentation for further detailled explanations), after that you will be able to add these devices to your personnal group.

Regards.
